src/main/java/athleticli/ui/Ui.java

```diff
@@ -0,0 +1,90 @@
+package athleticli.ui;
+
+import java.io.InputStream;
+import java.io.PrintStream;
+import java.util.Scanner;
+
+/**
+ * Defines the behavior of the CLI.
+ */
+public class Ui {
+    private static Ui uiInstance;
+    private final Scanner in;
+    private final PrintStream out;
+
+    /**
+     * Constructs a <code>Ui</code> object, whose input <code>in</code>
+     * and output <code>out</code> is the standard input and the standard
+     * output, respectively.
+     */
+    private Ui() {
+        this(System.in, System.out);
+    }
```

Comment:
There is no need for function overloading. You can just use the constructor below and pass in the `System.in` and `System.out` unless there is a very good reason why you need it